### PR TITLE
Add Nora to this weeks issue

### PR DESCRIPTION
**Pull request by SD10**
https://github.com/SD10/Nora

Nora is a Firebase abstraction layer for working with FirebaseDatabase and FirebaseStorage. This project was based off Moya for traditional networking. It makes working with Firebase easier by reducing boiler-plate code and preventing abstraction layer headaches.

### CODE REVIEW

File: Issues/Week177.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 * [Strongify](https://github.com/krzysztofzablocki/Strongify), by [@merowing_](https://twitter.com/merowing_)
 * [Gliding Collection](https://github.com/Ramotion/gliding-collection), by [@ramotion](https://twitter.com/ramotion)
 * [Get Schwifty](https://github.com/BenchR267/Get-Schwifty), by [@benchr](https://twitter.com/benchr)
+* [Nora](https://github.com/SD10/Nora), by SD10
```
Comment (RuiAAPeres) [inline Issues/Week177.md]: Please follow the same approach as the other PR's. (add a link your name + add yourself as a contributor)

Comment (SD10) [inline Issues/Week177.md]: @RuiAAPeres Sorry about that!

**Comment by dkhamsing** [comment]
week 177 is out, can u reopen this for week 178

**Comment by SD10** [comment]
@dkhamsing thank you 😂

**Pull request closed without merging**